src/create-parts-v2.js

```diff
@@ -52,6 +56,7 @@ const ls = db => {
 
 let lastUrl
 
+// get urls to process
```

Comment:
at this point they aren’t technically “urls”

“url” is the dynamo key name, but when a file is sliced the “url” is something like `::slice:{int}::{url}` and we actually have to skip over the primary key that matches that url because it just tells us that the data is sliced and how many slices it is.

you can see the code below checks `maxSize` before proceeding. the query itself avoids the main keys for sliced files but this acts as a sanity check in case there’s a bug in the query or the slicer.